Cartography/View.swift

```diff
@@ -10,19 +10,21 @@ import Foundation
 
 #if os(iOS) || os(tvOS)
     import UIKit
-    public typealias View = UIView
+    public typealias View = NSObject
```

Comment:
You look at the init function of NSLayoutConstraint. The param type is AnyObject, not UIView. This is Apple provided framework. And why is that so ? Because that's how Apple allows NSLayoutConstraint to work with LayoutGuide.